src/main/java/com/gmail/llmdlio/townyflight/TownyFlight.java

```diff
@@ -136,6 +137,7 @@ protected void unregisterEvents() {
 
 	private void registerCommands() {
 		getCommand("tfly").setExecutor(new TownyFlightCommand(this));
+		new TownyCommandIntegration;(this);

```

Comment:
The convention we have been using in our addon plugins is to give a more descriptive name to the addon command classes, this should probably be TownToggleFlightCommandAddon.